scarlet/observation.py

```diff
@@ -135,9 +134,10 @@ def render(self, model):
             `model` mapped into the observation frame
         """
 
-        image_model = model[self.slices]
         if self._diff_kernels is not None:
-            image_model = self._convolve(image_model)
+            image_model = self._convolve(model)[self.model_slices]
+        else:
+            image_model = model[self.model_slices]
```

Comment:
here

Reply:
If I'm reading this right, that only works if the model frame encompasses all the image pixel, which it would in the case of `coverage='union'` but not for `intersection`, right?

Reply:
No, on the contrary. The slices correspond to the size of the frame (I should name them frame_slices). In the tutorials, it works with both union and intersection.

Reply:
And it's a case where the union and intersection frame are not identical because the LowRes observation is large than the other.

Reply:
This has been dealt with in commit 75beb74